userguide/sharing.rst

```diff
@@ -971,6 +837,15 @@ some troubleshooting tips. Reading through this entire chapter before
 creating any SMB shares is recommended to gain a better understanding of
 the configuration scenario that meets the specific network requirements.
 
+`SMB Tips and Tricks <https://forums.freenas.org/index.php?resources/smb-tips-and-tricks.15/>`__
+shows helpful hints for configuring and managing SMB networking.
+The
+`FreeNAS and Samba (CIFS) permissions <https://www.youtube.com/watch?v=RxggaE935PM>`__
+and
+`Advanced Samba (CIFS) permissions on FreeNAS <https://www.youtube.com/watch?v=QhwOyLtArw0>`__
+videos clarify setting up permissions on SMB shares. Another helpful
+reference is
+`Methods For Fine-Tuning Samba Permissions <https://forums.freenas.org/index.php?threads/methods-for-fine-tuning-samba-permissions.50739/>`__.
 
 .. warning:: SMB1 is disabled by default for security. If legacy
```

Comment:
We should either remove this (my preference) or replace the instructions with just using the checkbox in the SMB service.